src/cne/elector.rs

```diff
@@ -17,6 +18,236 @@ pub struct Elector {
 
 impl From<String> for Elector {
   fn from(html_string: String) -> Self {
-    todo!()
+    let document = Html::parse_fragment(&html_string);
+    let selector = Selector::parse("table tbody tr td table tbody tr:last-child td table tbody tr:nth-child(2) td table:first-child tbody tr").unwrap();
```

Comment:
I think we could improve this by sanitizing the data before processing it.
As we now the response is a template, we could assume that for every response (of a successful search), between lines 1 and 43 (from your `html.html` file), there's no resources of our interest in order to build an `Elector`.

Then we could ignore those lines at first, that will help by making this `selector` much easier to use.

A good starting point could be:

```rust
    fn sanitize_html(text: String) -> String {
      let mut sanitized = String::default();

      text.split("\n").into_iter().enumerate().for_each(|(i, x)| {
        if i > 43 {
          sanitized.push_str(x);
        }
      });

      sanitized
    }
```